logs/logs.go

```diff
@@ -0,0 +1,100 @@
+package logs
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"github.com/deis/logger/storage"
+)
+
+const (
+	controllerPattern       = `^(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
+	controllerContainerName = "deis-controller"
+)
+
+var (
+	regex, _ = regexp.Compile(controllerPattern)
```

Comment:
not blocking for this PR - you can use [`regexp.MustCompile`](https://godoc.org/regexp#MustCompile) here, so that you don't have to bother with the error. might be good defensive programming